barnsbury/style-rtl.css

```diff
@@ -4390,3 +4390,32 @@ body.admin-bar .widget_eu_cookie_law_widget.widget.top {
 .wp-block-latest-posts .wp-block-latest-posts__post-full-content {
 	margin-top: 16px;
 }
+
+@media only screen and (max-width: 559px) {
+	.mobile-nav-side .has-menu .site-logo,
+	.mobile-nav-side .has-menu .site-title,
+	.mobile-nav-side .has-menu .site-description {
+		margin-left: 100px;
```

Comment:
I'm worried if this value will work if the button changes sizes. I'm not sure if other languages will change the width of the button.